### DISCUSSION TITLE
bug

### DISCUSSION THREAD

**Discussion by Hlypson** [Courseplay/Courseplay_FS22#3099] [General]
hey, I have a problem, when I turn on the mod, the game loads to 95% and stops when I checked the logs and the problem is with the BunkerSiloWrapper file
specifically 2024-02-12 00:00 C:/Users/proch/OneDrive/Dokumenty/My Games/FarmingSimulator2022/mods/FS22_Courseplay/scripts/silo/BunkerSiloWrapper.lua:268: attempt to index local 'area' (a nil value) what can I do to make the mod work?

**Comment by Tensuko** [comment]
-> Bugreport, nod in Discussion.

But already can tell you it should be a bunker silo build in your map.

**Comment by Hlypson** [comment]
I'm playing on the elmcreek map and I don't know what I need to do to make it work
